packages/app/src/resource/BotEditor.test.tsx

```diff
@@ -120,23 +120,17 @@ describe('BotEditor', () => {
   });
 
   test('Deploy error', async () => {
-    await setup('/Bot/123/editor');
-    expect(await screen.findByText('Deploy')).toBeInTheDocument();
+    const medplum = new MockClient();
+    medplum.router.router.add('POST', 'Bot/:id/$deploy', async () => [badRequest('Something went wrong')]);
 
-    // Mock the code frame
-    (screen.getByTestId<HTMLIFrameElement>('code-frame').contentWindow as Window).postMessage = (
-      _message: any,
-      _targetOrigin: any,
-      transfer?: Transferable[]
-    ) => {
-      (transfer?.[0] as MessagePort).postMessage({ error: badRequest('Error') });
-    };
+    await setup('/Bot/123/editor', medplum);
+    expect(await screen.findByText('Deploy')).toBeInTheDocument();
 
     await act(async () => {
       fireEvent.click(screen.getByText('Deploy'));
     });
 
-    expect(screen.getByText('Error')).toBeInTheDocument();
+    expect(screen.getByText('Something went wrong')).toBeInTheDocument();
```

Comment:
Changed this because common words like `Error` often appear in the document and give false positives on tests